CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## 1.1.12 - 2018-02-12
+This release is a backports changes from the master branch to the legacy branch
```

Comment:
I think you can remove "is a" from this line.

Reply:
woops